.github/workflows/publish.yml

```diff
@@ -374,18 +374,16 @@ jobs:
           curl -sSL "https://dd-package-tools.s3.amazonaws.com/dd-pkg/${DD_PKG_VERSION}/dd-pkg_Linux_x86_64.tar.gz" | tar -xz -C /usr/local/bin dd-pkg
       - name: Fix Git safe directories issue when in containers (actions/checkout#760)
         run: git config --global --add safe.directory /__w/vector/vector
-      - name: Checkout Vector
-        uses: actions/checkout@v4
-        with:
-          ref: ${{ inputs.git_ref }}
       - name: Download staged package artifacts (x86_64-unknown-linux-gnu)
         uses: actions/download-artifact@v3
         with:
           name: vector-${{ env.VECTOR_VERSION }}-x86_64-unknown-linux-gnu
           path: target/artifacts
       - name: Verify install of DEB package.
         run: |
-          ./scripts/verify-install.sh target/artifacts/vector_${{ env.VECTOR_VERSION }}-1_amd64.deb
+          curl -o verify-install.sh https://raw.githubusercontent.com/vectordotdev/vector/master/scripts/verify-install.sh
```

Comment:
I think this should be using the version of `verify-install.sh` that exists in whatever SHA is being built here. There is a (admittedly small) risk that the version on `master` will not be compatible with the version being released.

Reply:
Good point, `https://raw.githubusercontent.com/vectordotdev/vector/${{ inputs.git_ref }}/scripts/verify-install.sh` should do the job. But need to confirm this.